crypto/pkcs11/pkcs11helpers_test.go

```diff
@@ -133,6 +133,8 @@ module:
 	p11pubkeyfileobj.Uri.SetModuleDirectories(p11conf.ModuleDirectories)
 	p11pubkeyfileobj.Uri.SetAllowedModulePaths(p11conf.ModuleDirectories)
 
+	os.Setenv("OCICRYPT_OAEP_HASHALG", "sha1")
```

Comment:
question: would it make sense to add another tests with sha256 as well?

Reply:
a test with SoftHSM and sha256 doesn't work. See comments on related issue #60.

Reply:
https://github.com/containers/ocicrypt/issues/60#issuecomment-1111450382